system_meta/models.py

```diff
@@ -24,6 +24,10 @@ class IntegratedSystem(SafeDeleteModel, SoftDeleteActiveModel, TimestampedModel)
     description = models.TextField(blank=True)
     api_key = models.TextField(blank=True)
 
+    # Webhook URLs
+    sale_succeeded_webhook_url = models.URLField(blank=True, default="")
+    sale_refunded_webhook_url = models.URLField(blank=True, default="")
```

Comment:
I'd advocate for having a single webhook url and instead have us use a payload that includes an `event_type` discriminator field that has values like `order.success`, `order.refunded`, etc. I'd maybe even suggest namespacing it as `ecommerce.order.success` just for good measure. Having to add both a model field here and a new API url elsewhere for every event type won't scale well.

I think Stripe has a fairly straightforward way of structuring their webhook payloads: https://docs.stripe.com/webhooks (we obviously don't need all the fields they have at the root)